### PR TITLE
Contributing instructions in README

### PR DESCRIPTION
**Pull request by mauriceyap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,99 +1,174 @@
...
+# CCM Website
 
-Kick off your project with this default boilerplate. This starter ships with the main Gatsby configuration files you might need to get up and running blazing fast with the blazing fast app generator for React.
+This is the repo for [Christ Church Mayfair's website](https://christchurchmayfair.org).
 
-_Have another more specific idea? You may want to check out our vibrant collection of [official and community-created starters](https://www.gatsbyjs.org/docs/gatsby-starters/)._
+## Report a problem
 
-## 🚀 Quick start
+If you see something that doesn't look right, <del>text British Transport Police on 61016</del> [create an issue](https://github.com/ChristChurchMayfair/ccm-main-gatsby/issues/new/choose) or [email Tom Duckering](mailto:tom@christchurchmayfair.org).
```
Comment (bdrobinson) [inline README.md:7]: 😂

**Review by bdrobinson — APPROVED** [review]
This is *fantastic* – thanks for doing it! Really really clear and helpful.

File: README.md (hunk 1 above)
Comment (tomduckering) [reply README.md:7]: Cool - I'm happy to be the funnel.

**Review by tomduckering — APPROVED** [review]
Great - thanks for putting this together.

**Pull request merged**